HumanControls/Page.h

```diff
@@ -8,22 +8,32 @@
 class Page : DisplayInterface
 {
 public:
-    Page();
-    Page(int increment, int min, int max, bool isActive);
+    enum class PageName
```

Comment:
PageName is a bit of a misnomer. An enum isn't really a name.

Reply:
Renamed to PageType